### PR TITLE
Tighten up introduction lists.

### PR DESCRIPTION
**Pull request by timsifive**
Should help with Issue #62.

### CODE REVIEW

File: introduction.tex [hunk 1]
```diff
@@ -79,9 +79,9 @@ \section{Background}
 
 There are several use cases for dedicated debugging hardware, both
 internal to a CPU core and with an external connection.
-This specification defines techniques to support all of the use cases
-listed below. Some are optional to allow system designers to make
-cost vs capability tradeoffs.
+This specification address the the use cases listed below. Implementations can
```
Comment (mwachs5) [inline introduction.tex]: addresses

File: introduction.tex [hunk 2]
```diff
@@ -79,9 +79,9 @@ \section{Background}
...
+choose to not to implement every features, which means some use cases might not
```
Comment (mwachs5) [inline introduction.tex]: feature

File: introduction.tex [hunk 3]
```diff
@@ -140,30 +130,28 @@ \section{Supported Features}
...
-   \item Code can be downloaded efficiently.
    \item Each hart can be debugged from the very first instruction executed.
    \item A RISC-V hart can be halted when a software breakpoint instruction is
        executed.
-   \item Hardware can step over any instruction.
-   \item A RISC-V hart can be halted when a trigger matches the PC, read/write
-       address/data, or an instruction opcode.
-   \item Optional serial ports can be used for communication between debugger
-       and monitor, or as a general protocol between debugger and application.
+   \item Hardware single-step can execute one instruction at a time.
```
Comment (mwachs5) [inline introduction.tex]: Isn't this technically optional?

Comment (timsifive) [inline introduction.tex]: I believe single step is mandatory.

**Comment by timsifive** [comment]
Can I go ahead and merge this?

**Pull request merged**